CHANGELOG.md

```diff
@@ -19,6 +19,8 @@ $ cd gvm-tools && git log
 ### Added
 ### Changed
 
+- Fixed `send-targets.gmp.py` script because alive_test needs to be enum in create_target function.[#296](https://github.com/greenbone/gvm-tools/pull/295)
```

Comment:
```suggestion
- Fixed `send-targets.gmp.py` script, because alive_test needs to be from `AliveTest` enum in `create_target` function. [#296](https://github.com/greenbone/gvm-tools/pull/295)
```